src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -31,30 +31,30 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
         private IObjectDetailsViewer _detailsViewer;
         private readonly IServiceContainer _services;
         private readonly IRSettings _settings;
-        private string _title;
         private bool _deleted;
 
         public VariableViewModel() { Index = -1; }
 
+        public IRValueInfo Result {get;}
```

Comment:
What bug is this for?

Reply:
Just minor simplification. Partially undone.